test/loader.test.js

```diff
@@ -109,18 +111,15 @@ describe('loader.js', () => {
         });
 
         it('throws OpenError for non-existant file', async () => {
-            let thrownError;
```

Comment:
Changed weird way of checking rejection, to use [should.rejectedWith](https://shouldjs.github.io/#assertion-rejectedwith)